types/operator.go

```diff
@@ -71,6 +71,10 @@ func (o Operator) Validate() error {
 // or whatever an avs decides to use
 type Socket string
 
+func (s Socket) ToString() string {
```

Comment:
I think Go idiomatic is `String()` instead of `ToString()` which is more of Java convention.

Reply:
[8d91eb4](https://github.com/Layr-Labs/eigensdk-go/pull/256/commits/8d91eb48eb466d97cad93c4897ed6099ec751dff)